pipelines/src/main/groovy/testDoubles/ContextStub.groovy

```diff
@@ -64,4 +64,6 @@ class ContextStub {
     Configuration load(String s) {}
 
     String readFile(String s) {}
+
+    String ws(String s, Closure) {}
```

Comment:
ws -> workspace unless ws has special meaning :-)

Reply:
it has to be called ws